### ISSUE TITLE
Power pool accumulates rounding errors

### ISSUE THREAD

**Issue by troggy** [acebusters/acebusters-economy#55]
This line leads to rounding errors:
https://github.com/acebusters/economy/blob/fix/power-pool-precision/contracts/controller/MarketEnabled.sol#L72

Here is the test to reproduce the issue:
https://github.com/acebusters/economy/blob/fix/power-pool-precision/test/power.js#L159

```
Power pool: expected 100000833368057010 to equal 100000833361112030
```

**Comment by troggy** [comment]
just a 7 babz difference. on a second look it may be a bug in my test code

**Comment by bkcninja** [comment]
Resolved in commit 691caa5923043da978fd8deb05edec2140bbfbc7
